.github/workflows/build_cmake.yml

```diff
@@ -41,6 +41,12 @@ jobs:
             platform: linux_x64,
             cc: "gcc", cxx: "g++"
           }
+        - {
+            name: "Ubuntu 22.04 GCC", artifact: "Ubuntu-22.04-experimental",
```

Comment:
I would not add experimental in the name, because at the time of release we hopefully will have tested it. Any breakage will be easy to notice.

Reply:
I have an update mechanism that checks the archive name in the release and excludes experimental ones. And I would like to support only the latter for automatic updating. But for now I will not merge this, I am trying to understand whether we need 22.04 at all

Reply:
I think it's best if earliest Ubuntu possible is used for building. This is because sooner or later the plugin will start depending on some versioned glibc symbol that is available only on newer glibc and then users won't be able to use the plugin.

On Linux building with older libraries than on user's machine is generally fine, the opposite is not true.

Reply:
in that case ubuntu 22.04 should be main, because with 20.04 we already have builds error

Reply:
Yes.